### PR TITLE
update helm chart documentation about guac-viz deployment

### LINKED ISSUE #39: Documentation out of date: helm chart installs visualizer
The install docs [at this point](https://github.com/kusaridev/helm-charts/tree/main/charts/guac#guac-visualizer) say that the visualizer is not installed and that it needs to be run separately; however, there's a visualizer running in the cluster from a fresh helm install.

### PR DESCRIPTION
**Pull request by pxp928**
fixes #39

### CODE REVIEW

**Review by sunnyyip — APPROVED** [review]
LGTM

**Comment by sunnyyip** [comment]
You want to bump the version for the next release. 
https://github.com/kusaridev/helm-charts/blob/main/charts/guac/Chart.yaml#L12

**Comment by pxp928** [comment]
version updated

File: charts/guac/README.md [hunk 1]
```diff
@@ -37,6 +37,10 @@ The full GUAC component deployment is a set of asynchronous services that combin
 - **OSV Certifier**: This certifier gathers OSV vulnerability information from
   [osv.dev](https://osv.dev/) about packages.
 
+- **GUAC Visualizer**: The GUAC Visualizer is an experimental utility that can be used to interact with GUAC services. It acts as a way to visualize the software supply chain graph, as well as a means to explore the supply chain and prototype policies. 
+
+  **Note**: you will need to port-forward to the graphql service for the visualizer to work.
```
Comment (lee0c) [inline charts/guac/README.md:42]: is this still true? I thought this was only the case when running graphql in cluster and visualizer locally

Comment (pxp928) [reply charts/guac/README.md:42]: oh right, that would not be needed.

**Pull request merged**